src/key_bind.rs

```diff
@@ -75,5 +75,8 @@ pub fn key_binds() -> HashMap<KeyBind, Action> {
     bind!([Ctrl, Shift], Key::Named(Named::ArrowRight), PaneFocusRight);
     bind!([Ctrl, Shift], Key::Character("L".into()), PaneFocusRight);
 
+    // CTRL+Alt+L clears the scrollback.
+    bind!([Ctrl, Alt], Key::Character("L".into()), ClearScrollback);
```

Comment:
`L` because `CTRL` + `L` sends the VT100 sequence for clearing the terminal (but not the scrollback).